packages/elements/src/components/ino-markdown-editor/ino-markdown-editor.tsx

```diff
@@ -86,9 +93,7 @@ export class MarkdownEditor implements ComponentInterface {
   componentDidLoad(): void {
     this.createEditor();
     if (this.initialValue) {
-      this.editor.commands.setContent(this.markdownToHtml(), true);
-      this.textareaRef.value = this.htmlToMarkdown();
-      this.textareaRef.rows = this.textareaRef.value.split('\n').length;
+      this.handleInitialValueChange(this.initialValue);
```

Comment:
IMO it's kinda strange that we call a handler here. We should extract the content of `handleInitialValueChange` into another function (maybe `initializeEditor`) and call it.

Reply:
Dr. Strange was working here^^ 🌀 I´ll change it,